components/notebook-controller/config/overlays/openshift/params.env

```
@@ -0,0 +1 @@
+odh-kf-notebook-controller-image=quay.io/opendatahub/kubeflow-notebook-controller:1.7-35b81f5
```

is this correct, or should it be

```suggestion
odh-kf-notebook-controller-image=quay.io/opendatahub/kubeflow-notebook-controller:stable
```